README.md

```diff
@@ -232,3 +232,78 @@ package with features:
       -- code of the function
   end)
   ```
+
+## Tarantool 3.0 role
+
+`roles.expirationd` is a Tarantool 3.0 role for the expirationd
+package with features:
+
+* You can configure the expirationd role with `cfg` entry (check example).
+  cluster configuration allows to set the same parameters as
+  in [expirationd.cfg()](https://tarantool.github.io/expirationd/#cfg)
+* You can use persistent functions (i.e. created by `box.schema.func.create`).
```

Comment:
Please, proofread the documentation (for example, I use Google doc spellcheck).

> You can use persistent functions (i.e. created by box.schema.func.create). When configuring, role tries first to get a function from global namespace (_G) and if the function was not found then role tries to search in box.func for a function with the same name. If some functions from config are missing, expirationd will wait for their creation and start tasks when all of them are found. You can check alerts to see what functions are missing.

Reply:
changed